src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1130,7 +1130,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string? ba
                             }
                             else
                             {
-                                touchedFilesPath = unquoted;
+                                touchedFilesPath = ParseGenericPathToFile(unquoted, diagnostics, baseDirectory); ;
```

Comment:
Why didn't we need a similar change for VB?

Reply:
It already had quote handling etc.